apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html

```diff
@@ -89,7 +89,7 @@ <h2 class="fs-6">{% trans 'Initiator Organisations' %}</h2>
                                 aria-label="{% if organisation.title %}{{ organisation.title }}{% else %}{{ organisation.name}}{% endif %} {% trans 'Home' %}"
                               ></i>
                             {% endif %}
-                            <div class="ms-3 text-start">{% if organisation.title %}{{ organisation.title }}{% else %}{{ organisation.name}}{% endif %}</div>
+                            <div class="ms-3 text-start u-word-break-all">{% if organisation.title %}{{ organisation.title }}{% else %}{{ organisation.name}}{% endif %}</div>
```

Comment:
could we also use `u-break-word` instead of `u-word-break-all` as the latter will always break it unnaturally whereas `u-break-word` at least tries to brake the word at a natural point (or move it to a new line)

Reply:
I might misunderstand how `u-break-word` works, from what I understand it will:
1. try to break the word naturally
2. move it to a newline if 1 doesn't work
3. break it unnaturally if 1 and 2 fail

That should still prevent the overflow then?